public/app/plugins/panel/table/module.tsx

```diff
@@ -20,6 +20,7 @@ export const plugin = new PanelPlugin<Options, CustomFieldConfig>(TablePanel)
             max: 300,
           },
           shouldApply: () => true,
+          defaultValue: defaultPanelFieldConfig.width,
```

Comment:
for now, this approach works (ie passing in an explicit value for each field) but likely better if we can pass in the whole object once for fieldConfig/panelOptions and have the system set defaults for everything.  (@dprokop )

Reply:
Yes, I agree